src/System.Collections.NonGeneric/tests/Performance/Perf.HashTable.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+using Microsoft.Xunit.Performance;
+using System.Collections.Generic;
+
+namespace System.Collections.Tests
+{
+    public class Perf_HashTable : PerfTestBase
+    {
+        private IEnumerable<Hashtable> TestHashtables()
+        {
+            yield return CreateHashtable(100);
+            yield return CreateHashtable(1000);
+        }
+
+        private Hashtable CreateHashtable(int size)
+        {
+            Hashtable ht = new Hashtable();
+            for (int i = 0; i < size; i++)
+                ht.Add(CreateString(50), CreateString(50));
+            return ht;
+        }
+
+        [Benchmark]
+        public void ctor()
+        {
+            foreach (var iteration in Benchmark.Iterations)
+                using (iteration.StartMeasurement())
+                    new Hashtable();
```

Comment:
> you make a good point about the disposer that I hadn't considered

Sorry, I didn't actually mean Hashtable's Dispose (it doesn't have one).  I meant the IDisposable that's returned from StartMeasurement, just showing that there's not only the overhead of starting the measurement but also of stopping it.